napari/_qt/containers/_tests/test_qt_tree.py

```diff
@@ -148,3 +148,13 @@ def test_node_tree_view(qtbot):
     qsel.setCurrentIndex(QModelIndex(), qsel.Current)
     # check current in python
     assert root.selection._current is None
+
+
+def test_flags(tree_model):
+    """Some sanity checks on retrieving flags for nested items"""
+    assert not tree_model.hasIndex(5, 0, tree_model.index(1))
+    last = tree_model._root.pop()
+    tree_model._root[1].append(last)
+    assert tree_model.hasIndex(5, 0, tree_model.index(1))
+    idx = tree_model.index(5, 0, tree_model.index(1))
+    assert bool(tree_model.flags(idx) & Qt.ItemIsEnabled)
```

Comment:
Very minor, but consistent with other changes here.

```suggestion
    assert bool(tree_model.flags(idx) & Qt.ItemFlag.ItemIsEnabled)
```